frontend/public/components/nav.jsx

```diff
@@ -376,20 +371,17 @@ export class Nav extends React.Component {
             }
             <HrefLink href="/overview" name="Status" activePath="/overview/" onClick={this.close} required={FLAGS.OPENSHIFT} />
             <HrefLink href="/status" name="Status" activePath="/status/" onClick={this.close} disallowed={FLAGS.OPENSHIFT} />
-            <HrefLink href="/catalog" name="Catalog" activePath="/catalog/" onClick={this.close} />
             <HrefLink href="/search" name="Search" onClick={this.close} startsWith={searchStartsWith} />
             <ResourceNSLink resource="events" name="Events" onClick={this.close} />
           </NavSection>
-
-          <NavSection required={FLAGS.OPERATOR_LIFECYCLE_MANAGER} text="Operators" img={operatorImg} activeImg={operatorActiveImg} >
-            {
-              // <HrefLink required={FLAGS.KUBERNETES_MARKETPLACE} href="/marketplace" name="Kubernetes Marketplace" activePath="/marketplace/" onClick={this.close} />
-            }
-            <ResourceNSLink model={ClusterServiceVersionModel} resource={ClusterServiceVersionModel.plural} name="Cluster Service Versions" onClick={this.close} />
+          <NavSection text="Catalog" icon="pficon pficon-catalog">
```

Comment:
Should this be plural?  "Catalogs"

Reply:
> Should this be plural? "Catalogs"
Don't think so cuz it's only a single catalog :)  it's essentially "everything catalog ... ":D